content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -256,14 +256,60 @@ spec:
       armory:
         terraform:
           enabled: true
+          # Remote backends is an Early Access feature. Do not use it in production. Omit this field if you do not want to use remote backends.
+          remoteBackendSupport: true
+    profiles:
+      deck:
+        # Enables the UI for the Terraform Integration stage
+        settings-local.js: |
+          window.spinnakerSettings.feature.terraform = true;
 ```
 
+This example manifest also enables the Terraform Integration UI and support for remote backends.
+
 **Halyard**
 
 ```
 hal armory terraform enable
 ```
 
+### Remote backends
+
+{{< include "experimental-feature.html" >}}
+
+This Early Access feature is available in Armory 2.AB.XY and later.
```

Comment:
@ethanfrogers What version should we tell people this experiment is available in? Or is it in a later stage than experiment?

Reply:
@aelath

Reply:
@brian-armory Yeah, I think the approach is to tell people to contact us.